plugins/aglight/export.js

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable radix */
```

Comment:
Why did you disable this rule? It can be easily silenced by passing the second (radix) parameter to `parseInt` to avoid nasty unexpected bugs: https://eslint.org/docs/rules/radix

```
parseInt(someString, 10)
```